optuna_dashboard/_serializer.py

```diff
@@ -116,25 +116,20 @@ def serialize_attrs(attrs: dict[str, Any]) -> list[Attribute]:
     return serialized
 
 
-def serialize_study_summary(summary: StudySummary) -> dict[str, Any]:
+def serialize_frozen_study(study: FrozenStudy) -> dict[str, Any]:
     serialized = {
-        "study_id": summary._study_id,
-        "study_name": summary.study_name,
-        "directions": [d.name.lower() for d in summary.directions],
-        "user_attrs": serialize_attrs(summary.user_attrs),
-        "is_preferential": getattr(summary, "_system_attrs", {}).get(
-            _SYSTEM_ATTR_PREFERENTIAL_STUDY, False
-        ),
+        "study_id": study._study_id,
+        "study_name": study.study_name,
+        "directions": [d.name.lower() for d in study.directions],
+        "user_attrs": serialize_attrs(study.user_attrs),
+        "is_preferential": study.system_attrs.get(_SYSTEM_ATTR_PREFERENTIAL_STUDY, False),
     }
 
-    if summary.datetime_start is not None:
-        serialized["datetime_start"] = summary.datetime_start.isoformat()
```

Comment:
To add `datetime_start`, I need to add this attribute in `FrozenStudy`.

Reply:
I'would say we should drop `datetime_start` for performance reasons. Could you remove a following line as well?

https://github.com/optuna/optuna-dashboard/blob/09f61ce23c1d004d9fc4711c953fd31320166c6a/optuna_dashboard/ts/types/index.d.ts#L126